### PR TITLE
Emscripten python live drawing

### PR DESCRIPTION
**Pull request by Ecco**
Allow long-running Python scripts to refresh the display

### CODE REVIEW

File: python/port/helpers.cpp [hunk 1]
```diff
@@ -4,15 +4,23 @@ extern "C" {
...
+  /* This function is called very frequently by the MicroPython engine. We grab
+   * this opportunity to interrupt execution and/or refresh the display on
+   * platforms that need it. */
+
+  /* Doing too many things here slows down Python execution quite a lot. So we
+   * only do things once in a while and return as soon as possible otherwise. */
   static int c = 0;
   c++;
-  if (c%20000 != 0) {
+  if (c % 20000 != 0) {
```
Comment (boricj) [inline python/port/helpers.cpp]: Might as well replace this with something a bit cleaner:
```c
static int c;

c = (c + 1) % 20000;
if (c != 0) {
  return;
}
```

Comment (Ecco) [reply python/port/helpers.cpp]: I'm sorry, but dumb me doesn't understand what's different in what you suggest 😄

Comment (boricj) [reply python/port/helpers.cpp]: The behavior is (nearly) identical, it's just more concise and a bit easier to read. It's probably just a matter of personal taste though.

Comment (LeaNumworks) [reply python/port/helpers.cpp]: Thanks your for the suggestion! I made a commit cleaning this piece of code.

**Pull request merged**